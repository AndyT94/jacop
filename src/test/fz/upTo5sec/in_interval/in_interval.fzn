var 3..3: v:: output_var;
var 2..2: low:: output_var;
var 5..5: up:: output_var;
solve  satisfy;
