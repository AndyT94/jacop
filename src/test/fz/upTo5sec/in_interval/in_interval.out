v = 3;
low = 2;
up = 5;
----------